Model/Quote/Checkout.php

```diff
@@ -5,14 +5,15 @@
 use Magento\Checkout\Model\Cart;
 use Magento\Checkout\Model\Session;
 use MyParcelNL\Magento\Helper\Data;
+use MyParcelNL\Magento\Model\Checkout\Carrier;
 use MyParcelNL\Magento\Model\Sales\Repository\PackageRepository;
 use \Magento\Store\Model\StoreManagerInterface;
 
 class Checkout
 {
-    const selectCarriersArray = 0;
-    const selectCarrierPath   = 1;
-    const platform            = 'myparcel';
+    const SELECT_CARRIER_ARRAY = 0;
+    const SELECT_CARRIER_PATH  = 1;
```

Comment:
In deze gevallen zijn de consts alleen maar verwarrender vind ik, het lijkt nu in de code alsof je echt specifieke indexes pakt terwijl het niet zo spannend is

Reply:
Volgens de [PHP standaarden](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-1-basic-coding-standard.md#4-class-constants-properties-and-methods) die we volgen moeten alle consts met upper case worden beschreven.

Reply:
dat is niet wat ik bedoel, ik bedoel dat ik hier die consts niet nodig vind omdat het de plekken waar ze gebruikt worden alleen maar moeilijker leesbaar maakt, kan je geen classje maken van die array waardoor het netter kan?